src/main/java/gregtech/client/renderer/texture/Textures.java

```diff
@@ -241,6 +242,13 @@
     public static final SimpleOverlayRenderer COVER_INTERFACE_PROXY = new SimpleOverlayRenderer("cover/cover_interface_proxy");
     public static final SimpleOverlayRenderer COVER_INTERFACE_WIRELESS = new SimpleOverlayRenderer("cover/cover_interface_wireless");
 
+    public static final SimpleOverlayRenderer SURFACE_ROCK_TEXTURE = new SimpleOverlayRenderer("stones/surface_rock_stone");
```

Comment:
Did this texture addition sneak through various rebases?

Reply:
It seems it did, although more accurately, it was never removed; it was used in the previous surface rock model.